### PR TITLE
fix: implement interfaces in Android sources

### PR DESCRIPTION
**Pull request by WoLewicki**
PR fixing not implemented interfaces in `Source` components, adding not implemented setters for `existing` and removing unused `images` and `nativeImages` props

### CODE REVIEW

**Comment by WoLewicki** [comment]
@mfazekas is there a reason on some `Source` components, the `existing` prop was not implemented in managers?

**Comment by mfazekas** [comment]
> @mfazekas is there a reason on some `Source` components, the `existing` prop was not implemented in managers?

It was an overlook, `existing`  should be there for other sources. Just that are mostly used in practice for ShapeSource.

File: android/src/main/java/com/rnmapbox/rnmbx/components/styles/sources/RNMBXRasterDemSourceManager.kt [hunk 1]
```diff
@@ -27,4 +30,14 @@ class RNMBXRasterDemSourceManager(private val mContext: ReactApplicationContext)
...
     }
+
+    @ReactProp(name = "existing")
+    override fun setExisting(view: RNMBXRasterDemSource, value: Dynamic) {
+        view.mExisting = value.asBoolean()
+    }
+
+    @ReactProp(name = "tileSize")
+    override fun setTileSize(view: RNMBXRasterDemSource, value: Dynamic) {
+        // TODO: it does not exist on this class
```
Comment (mfazekas) [inline android/src/main/java/com/rnmapbox/rnmbx/components/styles/sources/RNMBXRasterDemSourceManager.kt:41]: we can log an error, that it's not implemented.
```suggestion
       Logger.e("RNMBXRasterDemSourceManager", "tileSize not implemented")
```

**Review by mfazekas — APPROVED** [review]

**Comment by WoLewicki** [comment]
@mfazekas this one also looks ready now 🚀

**Pull request merged**